protos/flyteidl/event/event.proto

```diff
@@ -95,6 +95,10 @@ message NodeExecutionEvent {
 
     // Whether this node yielded a dynamic workflow.
     bool is_dynamic = 18;
+
+    // String location uniquely identifying where the deck HTML file is
+    // NativeUrl specifies the url in the format of the configured storage provider (e.g. s3://my-bucket/randomstring/suffix.tar)
+    string deck_uri = 19;
```

Comment:
Should this not be per task? What if the task retries had some decks as well

Reply:
are you thinking we will need to visualize these even if the task fails?

Reply:
Atleast maintain the flexibility to do so